polaris/ocean/convergence/spherical/analysis.py

```diff
@@ -0,0 +1,332 @@
+import datetime
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from polaris import Step
+from polaris.ocean.resolution import resolution_to_subdir
+
+
+class SphericalConvergenceAnalysis(Step):
+    """
+    A step for analyzing the output from the geostrophic convergence test
+
+    Attributes
+    ----------
+    resolutions : list of float
+        The resolutions of the meshes that have been run
+
+    icosahedral : bool
+        Whether to use icosahedral, as opposed to less regular, JIGSAW
+        meshes
+
+    dependencies_dict : dict of dict of polaris.Steps
+        The dependencies of this step
+    """
+    def __init__(self, component, resolutions, icosahedral, subdir,
+                 dependencies, convergence_vars):
```

Comment:
`convergence_vars` needs to be added to the docstring for both `Parameters` here and `Attributes` of the class.  It will need quite a bit of explanation, just like `dependencies`.